appservice/src/deploy/runWithZipStream.ts

```diff
@@ -71,14 +71,21 @@ const commonGlobSettings: {} = {
 /**
  * Adds files using glob filtering
  */
-async function getFilesFromGlob(folderPath: string): Promise<string[]> {
+async function getFilesFromGlob(folderPath: string, client: SiteClient): Promise<string[]> {
     const zipDeployConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(ext.prefix, vscode.Uri.file(folderPath));
     const globOptions = { cwd: folderPath, followSymbolicLinks: true, dot: true };
     const globPattern: string = zipDeployConfig.get<string>('zipGlobPattern') || '**/*';
     const filesToInclude: string[] = await globby(globPattern, globOptions);
+    const zipIgnorePatternStr = 'zipIgnorePattern';
 
-    const ignorePattern: string | string[] = zipDeployConfig.get<string | string[]>('zipIgnorePattern') || '';
-    const filesToIgnore: string[] = await globby(ignorePattern, globOptions);
+    const ignorePatternList: string | string[] = zipDeployConfig.get<string | string[]>(zipIgnorePatternStr) || '';
+    const filesToIgnore: string[] = await globby(ignorePatternList, globOptions);
+
+    ext.outputChannel.appendLog(localize('zipIgnoreFileMsg', `Ignoring files from \"{0}.{1}\"`, ext.prefix, zipIgnorePatternStr), { resourceName: client.fullName });
+
+    for (const pattern of ignorePatternList) {
```

Comment:
Can you double check what happens if `ignorePatternList` is a `string` instead of `string[]`. You might need a little extra logic to fix that case

Reply:
Thank you! I missed that part